sig-cluster-lifecycle/README.md

```diff
@@ -25,7 +25,7 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=ljK5dgSA7
 * [Slack](https://kubernetes.slack.com/messages/sig-cluster-lifecycle)
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
 
-## GitHub Teams
+## GitHub Teams
```

Comment:
When I click the view button, here it shows as `## GitHub Teams`. It shows okay as a sub-title on the master branch. So I think you need to use what on the master branch here :)

Reply:
Not sure how those got updated, will change.